src/styles/styles.css

```diff
@@ -82,3 +126,63 @@
     font-size: 1rem;
     padding: 0.25em 0.5em;
 }
+
+/* ********  GAME-OVER-PAGE  ******** */
+
+#game-over-page {
+    /* visibility: hidden; */
```

Comment:
I think this is not meant to be commented, as both the gameover screen and start screen are showing if you scroll down.

Reply:
fixed